README.md

```diff
@@ -18,7 +18,7 @@ Before setting up the Titanium SDK, you must generate the appropriate credential
     ```xml
     <modules>
       <module platform="iphone" version="1.7.0">com.williamrijksen.onesignal</module>
-      <module platform="android" version="1.7.0">com.williamrijksen.onesignal</module>
```

Comment:
We should bump the iPhone version as well, because of the automatic building of the module(s).